test/unittests/utils/test_connection_live.py

```diff
@@ -348,9 +290,9 @@ def test_log_request() -> None:
     expected_output = {
         "method": "GET",
         "url": "http://example.com/",
-        "headers": {"Authorization": "Bearer my-ve[+13]", "request-header": "request-value"},
+        "headers": {"Authorization": "Bearer ***", "request-header": "request-value"},
         "timeout": 1,
-        "data": {"password": "************************", "foo": "bar"},
+        "data": {"password": "***", "foo": "bar"},
```

Comment:
would the password still be in the request of connection? is that not the point to remove it?

Reply:
I was confused by that too: This is needed for the `create` command, when creating new users, because there we of course also have the password in the payload.  
I would have preferred not to have to do that check - but I suppose that would mean having a separate connection for the `create` command... that might actually be a good idea, but is beyond the scope of what I wanted to do here.